website/content/docs/oss/configuration/worker/pki-worker.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: PKI Worker Configuration
+description: |-
+  PKI worker-specific parameters.
+---
+
+
+## PKI Worker Configuration
+PKI Workers authenticate to Boundary using a new certificate-based method, allowing for worker deployment
+without using a shared KMS. PKI Workers can also be configured using CLI commands.
```

Comment:
KMS ones will be able to as well for API tags...we could update the text later, but it might be better to just state that for PKI workers the name and description are configured via the API.

Reply:
Ah, OK -- you already have that info. I'd just remove that last sentence about CLI commands, personally.

Reply:
removed 👍